### PR TITLE
Grey/pipeline job to job

### PR DESCRIPTION
**Pull request by Tryneus**
Essentially a revert of #6039 now that we have a better name for `pfs.Job` (`Commitset`).

### CODE REVIEW

**Review by PFedak — CHANGES_REQUESTED** [review]
looks like there are a few dozen occurrences of "pipeline job" which should be fixed up

File: src/internal/collection/etcd_collection_test.go [hunk 1]
```diff
@@ -82,134 +82,134 @@ func TestDeletePrefix(t *testing.T) {
 	env := testetcd.NewEnv(t)
 	uuidPrefix := uuid.NewWithoutDashes()
 
-	pipelineJobInfos := col.NewEtcdCollection(env.EtcdClient, uuidPrefix, nil, &pps.PipelineJobInfo{}, nil, nil)
+	jobInfos := col.NewEtcdCollection(env.EtcdClient, uuidPrefix, nil, &pps.JobInfo{}, nil, nil)
 
-	pj1 := &pps.PipelineJobInfo{
-		PipelineJob: client.NewPipelineJob("prefix/suffix/pipelineJob"),
-		Pipeline:    client.NewPipeline("p"),
+	pj1 := &pps.JobInfo{
```
[the remark this answers is not in the document]
Comment (Tryneus) [reply src/internal/collection/etcd_collection_test.go]: good catch - fixed

File: src/server/pps/pps.go [hunk 2]
```diff
@@ -8,23 +8,23 @@ import (
...
-	PipelineJob *pps.PipelineJob
+// ErrJobFinished represents a finished job error.
+type ErrJobFinished struct {
+	Job *pps.Job
 }
 
-func (e ErrPipelineJobFinished) Error() string {
-	return fmt.Sprintf("pipeline job %v has already finished", e.PipelineJob.ID)
+func (e ErrJobFinished) Error() string {
+	return fmt.Sprintf("pipeline job %v has already finished", e.Job.ID)
```
[the remark this answers is not in the document]
Comment (Tryneus) [reply src/server/pps/pps.go]: got 'em

**Review by PFedak — APPROVED** [review]
LGTM

**Pull request merged**